assembl/tests/views_tests/test_routes.py

```diff
@@ -230,9 +230,17 @@ def test_route_discussion_post_legacy(discussion, root_post_1, test_app):
     from urllib import quote_plus
     # Encode the URL so that it is compatible with URLs
     url_post_id = quote_plus(root_post_1.uri())
-    route = discussion_route(slug, "posts", url_post_id)
+    route = discussion_route(slug, "posts", url_post_id, backbone=True)
     resp = test_app.get(route)
-    assert resp.status_int == 303
+    assert resp.status_int == 200
+
+    from assembl.lib.frontend_urls import FrontendUrls
+    headers = get_response_headers(resp)
+    redirect_url = unquote(headers['Location'])
+    print redirect_url
```

Comment:
Can you remove the print in test?

Reply:
+1 and there are serveral occurences of print